test/e2e/etcdbackupserver_e2e_test.go

```diff
@@ -0,0 +1,147 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	configv1alpha1 "github.com/openshift/api/config/v1alpha1"
+	"github.com/openshift/cluster-etcd-operator/test/e2e/framework"
+	"github.com/openshift/library-go/test/library"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apiserver/pkg/storage/names"
+	"k8s.io/klog/v2"
+	"k8s.io/utils/pointer"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	defaultBackupCRName = "default"
+	testSchedule        = "*/5 * * * *"
+	testTimeZone        = "UTC"
+	backupHostPath      = "/var/backup/etcd"
+)
+
+func TestEtcdBackupServer(t *testing.T) {
+	configClient := framework.NewConfigClient(t)
+	clientSet := framework.NewClientSet("")
+	etcdPodsClient := clientSet.CoreV1Interface.Pods(operandNameSpace)
+
+	backupCR := createDefaultBackupCR(testSchedule, testTimeZone)
+	_, err := configClient.Backups().Create(context.Background(), backupCR, metav1.CreateOptions{})
+	require.NoError(t, err)
+
+	err = library.WaitForPodsToStabilizeOnTheSameRevision(t, etcdPodsClient, etcdLabelSelector, 5, 1*time.Minute, 5*time.Second, 30*time.Minute)
+	require.NoError(t, err)
+
+	masterNodes, err := clientSet.CoreV1Interface.Nodes().List(context.Background(), metav1.ListOptions{LabelSelector: masterNodeLabel})
+	require.NoErrorf(t, err, "error listing master nodes")
+
+	for _, node := range masterNodes.Items {
+		err = isBackupExist(t, node)
+		require.NoError(t, err)
+	}
+
+	t.Cleanup(func() {
+		cErr := configClient.Backups().Delete(context.Background(), backupCR.Name, metav1.DeleteOptions{})
+		require.NoError(t, cErr)
+	})
+}
+
+func createDefaultBackupCR(schedule, timeZone string) *configv1alpha1.Backup {
+	return &configv1alpha1.Backup{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: defaultBackupCRName,
+			Annotations: map[string]string{
+				"default": "true",
+			},
+		},
+		Spec: configv1alpha1.BackupSpec{
+			EtcdBackupSpec: configv1alpha1.EtcdBackupSpec{
+				Schedule: schedule,
+				TimeZone: timeZone,
+				RetentionPolicy: configv1alpha1.RetentionPolicy{
```

Comment:
@tjungblu  As `RetentionPolicy` is `optional`, initially I have omitted it

however, I received a `validation error` 

The CR I used during manual testing has no retention at all and there was no error 

What am I missing ?